Sources/SentryCrash/Recording/Tools/SentryCrashObjC.c

```diff
@@ -232,7 +232,7 @@ decodeIsaPointer(const void *const isaPointer)
 #if ISA_TAG_MASK
     uintptr_t isa = (uintptr_t)isaPointer;
     if (isa & ISA_TAG_MASK) {
-#    if defined(__arm64__)
+#    if TARGET_OS_IOS
```

Comment:
`h`: Travis failed because [`ISA_MASK_OLD` only exists for `__arm64__`](https://github.com/getsentry/sentry-cocoa/blob/master/Sources/SentryCrash/Recording/Tools/SentryCrashObjCApple.h#L52). Some iPhones with the A6 and A6X chip use armv7s. Can we please change it to the following:
```suggestion
#    if TARGET_OS_IOS && defined(__arm64__)
```